src/reference/environments/live-full.yml

```diff
@@ -0,0 +1,12 @@
+name: cylc-8-live-with-rose
+# Cylc 8, the UI and UI server, Cylc Rose and Metomi Rose
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9
+  - pip
+  - pip:
+    - git+https://github.com/cylc/cylc-flow.git
+    - git+https://github.com/cylc/cylc-uiserver.git
+    - git+https://github.com/metomi/rose.git
+    - git+https://github.com/cylc/cylc-rose.git
```

Comment:
This kinda works, however, doesn't install non-python things like the Configurable HTTP Proxy needed for JupyterHub.